nservicebus/messaging/publish-subscribe/index.md

```diff
@@ -48,20 +48,17 @@ Available subscription persistences include
  * [InMemory](/nservicebus/persistence/in-memory.md)
  * [Azure Storage](/nservicebus/azure-storage-persistence)
 
-Transports that require persistences
-
- * [MSMQ](/nservicebus/msmq)
- * [SQLServer](/nservicebus/sqlserver)
- * [Azure Storage Queues](/nservicebus/azure-storage-queues)
-
+The emulated publish-subscribe is used by the unicast transports ({include:transports-unicast}).
```

Comment:
> unicast transports

I believe this is the first time "unicast" is used in this doco. Maybe it's not necessary? If necessary, we need to explain it first.